hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcConnection.java

```diff
@@ -257,6 +258,7 @@ private void connect() {
         .option(ChannelOption.TCP_NODELAY, rpcClient.isTcpNoDelay())
         .option(ChannelOption.SO_KEEPALIVE, rpcClient.tcpKeepAlive)
         .option(ChannelOption.CONNECT_TIMEOUT_MILLIS, rpcClient.connectTO)
+        .option(ChannelOption.WRITE_BUFFER_WATER_MARK, new WriteBufferWaterMark(rpcClient.bufferLowWatermark, rpcClient.bufferHighWatermark))
```

Comment:
Create a WriteBufferWaterMark and store it in NettyRpcClient so we do not need to create it everytime?

Reply:
> Create a WriteBufferWaterMark and store it in NettyRpcClient so we do not need to create it everytime?

There is really no need to create a WriteBufferWaterMark every time.I will modify this definition in NettyRpcConnection.